CedarJava/src/main/java/com/cedarpolicy/model/schema/Schema.java

```diff
@@ -21,86 +21,127 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
-import java.io.IOException;
-import java.util.Objects;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.Optional;
 
 /** Represents a schema. */
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
what does this do?

Reply:
So that if `schemaJson` is empty, then instead of serializing it as `"json": null` (which makes the Rust parsers angry), Jackson ignores it.